lib/build.js

```diff
@@ -1,86 +1,123 @@
-/* jshint node: true */
-
-module.exports = function (opts, callback) {
-  'use strict';
-
-  // Usage:
-  // require('modernizr').build(config, callback);
-  //
-  // Where:
-  // config - A JSON object of config options (see ./lib/config-all.json)
-  // callback - A callback function once build completes
-
-  opts = opts || {};
-
-  var cwd = process.cwd();
-  var path = require('path');
-  var spawn = require('win-spawn');
-  var fs = require('fs');
-
-  var localRoot = path.join(__dirname, '..');
-
-  // Temporarily change working dir to Modernizr root
-  process.chdir(localRoot);
-
-  // Context-sensitive requires
-  var grunt = require('grunt');
-
-  // So we don't make path assumptions, we load the current Gruntfile
-  require(path.join(localRoot, 'Gruntfile'))(grunt);
-
-  // and store our config options for later use.
-  var settings = grunt.config();
-
-  // White noise suppression
-  var verbose = (opts.verbose !== false);
-  delete opts.verbose;
+// this file configures require.js based on enviroment
+'use strict';
+
+var inBrowser = typeof define == 'function' && typeof define.amd == 'object';
+if (!inBrowser) {
+  // Defines a module that works in Node and AMD.
+  // http://git.io/bHfJ
+  var define = function (name, factory) {
+    module.exports = factory(require, exports, module);
+  };
+}
+
+var requireConfig = {
+  optimize: 'none',
+  optimizeCss: 'none',
+  useStrict: true,
+  include: ['modernizr-init'],
+  fileExclusionRegExp: /^(.git|node_modules|modulizr|media|test)$/,
+  wrap: {
+    start: '\n;(function(window, document, undefined){',
+    end: '})(this, document);'
```

Comment:
There were some requests that we switch our outer wrapper to just use `window` since it absolutely requires a real `window`.